components/imageCard.js

```diff
@@ -1,45 +1,65 @@
 import {useState, useEffect} from "react";
-import Image from "next/image";
+import Image from "next/legacy/image";
 import styled from "styled-components";
+import deleteIcon from "../public/deleteIcon.svg";
+import deleteIconFilled from "../public/deleteIconFilled.svg";
+import likeIcon from "../public/likeIcon.svg";
+import likeIconFilled from "../public/likeIconFilled.svg";
+import {useContext} from "react";
+import {ImageContext} from "./ImageContext";
 
 export default function ImageCard() {
+  // const {setFavorite} = useContext(ImageContext);
```

Comment:
Please delete code you don't use anymore.

Reply:
Done.